jenkins_jobs/build_host_os/script.sh

```diff
@@ -0,0 +1,63 @@
+GSA_HOST_NAME="pokgsa.ibm.com"
+GSA_BUILDS_DIR="/gsa/pokgsa/projects/h/host-os/to_build"
+VERSIONS_REPO_DIR="components"
+REPO_FILE="extras/centOS/7.2/mock/epel-7-ppc64le.cfg"
+MAIN_REPO_URL_PREFIX="mirror.centos.org/altarch"
+INTERNAL_REPO_MIRROR_URL_PREFIX="mirror.aus.stglabs.ibm.com/centos-altarch"
+
+function on_exit(){
+    rsync -e ssh --verbose --compress --stats --times --chmod=a+rwx,g+rwx,o- "STATUS" "maurosmr@${GSA_HOST_NAME}:${GSA_BUILDS_DIR}/${BUILD_NUMBER}/"
```

Comment:
Username `maurosmr` is hard-coded here.  Variables should not be prefixed with `GSA_`.